shims/mapr510/impl/pom.xml

```diff
@@ -11,6 +11,7 @@
   <artifactId>pentaho-hadoop-shims-mapr510</artifactId>
   <version>8.0-SNAPSHOT</version>
   <properties>
+    <org.xerial.snappy.version>1.1.1.3</org.xerial.snappy.version>
```

Comment:
if we define the same version of snappy for all shims, let's define it in parent shims/pom.xml
like 
https://github.com/pentaho/pentaho-hadoop-shims/blob/45aacc296be16983fac9291d01c65060e8641c95/pom.xml#L141
https://github.com/SergeyTravin/pentaho-hadoop-shims/blob/8c74944ac559af24ca84f36b003dd6534dfbf1f2/shims/cdh511/impl/pom.xml#L77